AzureCalling/Controllers/CallViewController.swift

```diff
@@ -151,6 +152,9 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
 
     private func openAudioDeviceDrawer() {
         let audioDeviceSelectionDataSource = AudioDeviceSelectionDataSource()
+        audioDeviceSelectionDataSource.didSelectAudioDevice = {
+            self.updateAudioDeviceButtonIcon()
```

Comment:
Let's use [weak self] here  which prevent retain cycle
audioDeviceSelectionDataSource.didSelectAudioDevice = { [weak self] in 
guard let self = self else {return}
            self.updateAudioDeviceButtonIcon()